doc/user_guide/user_guide.md

```diff
@@ -590,6 +590,30 @@ INTO SCRIPT CLOUD_STORAGE_EXTENSION.EXPORT_PATH WITH
   PARALLELISM     = 'iproc(), floor(random()*<MULTIPLIER>)';
 ```
 
+### S3 Path Style Access
+
+Amazon S3 [deprecated the path style access][s3-path-style-deprecation] to the
+buckets at the end of the 2020. This breaks the access to the bucket that
+contain dot (`.`) in their names.
+
+[s3-path-style-deprecation]: https://forums.aws.amazon.com/ann.jspa?annID=6776
+
+To enable the path style access to the bucket, you can set the
+`S3_PATH_STYLE_ACCESS` parameter to `true`.
```

Comment:
I would add this as a note to the Summary of the release letter for better visibility

Reply:
And also a note about renaming scripts